packages/block-library/src/search/edit.js

```diff
@@ -408,10 +408,24 @@ export default function SearchEdit( {
 			</BlockControls>
 
 			<InspectorControls>
-				<PanelBody title={ __( 'Settings' ) }>
-					<VStack
-						className="wp-block-search__inspector-controls"
-						spacing={ 4 }
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							width: undefined,
+							widthUnit: undefined,
+						} );
+					} }
```

Comment:
It would be a good idea to add the `dropdownMenuProps` here to ensure that the drop-down menu is positioned on the left ([Example](https://github.com/WordPress/gutenberg/blob/2397525dfbe29814ace66dd1dbca152208aad9a8/packages/block-library/src/button/edit.js#L127)).